src/features/api/utils.ts

```diff
@@ -1,7 +1,16 @@
+import { GithubRegistry } from '@hyperlane-xyz/registry';
+import { MultiProvider } from '@hyperlane-xyz/sdk';
+
 export function successResult<R>(data: R): { success: true; data: R } {
   return { success: true, data };
 }
 
 export function failureResult(error: string): { success: false; error: string } {
   return { success: false, error };
 }
+
+export async function getMultiProvider(): Promise<MultiProvider> {
+  const registry = new GithubRegistry();
```

Comment:
might be nice to consolidate this instance with the store instance so we can override the registry url in one place

Reply:
Agreed though there's some trickiness because stores are meant to be consumed via react hooks whereas this is server-side code. Leaving a TODO comment for now.